test/functional/pdo_sqlsrv/pdo_140_emulate_prepare_mix_binary.phpt

```diff
@@ -4,22 +4,23 @@ Test emulate prepare with mix bound param encodings including binary data
 <?php require('skipif_mid-refactor.inc'); ?>
 --FILE--
 <?php
-class MyStatement extends PDOStatement {
-  public function BindValues(array &$values, $placeholder_prefix, $columnInformation) {
-    $max_placeholder = 0;
-    foreach ($values as $field_name => &$field_value) {
-      $placeholder = $placeholder_prefix . $max_placeholder++;
-      if (isset($columnInformation['blobs'][$field_name])) {
-        $this->bindParam($placeholder, $field_value, PDO::PARAM_LOB, 0, PDO::SQLSRV_ENCODING_BINARY);
-      }
-      else {
-        // Even though not a blob, make sure we retain a copy of these values.
-        $this->bindParam($placeholder, $field_value, PDO::PARAM_STR);
-      }
+class MyStatement extends PDOStatement
+{
+    public function BindValues(array &$values, $placeholder_prefix, $columnInformation)
```

Comment:
Looks like you only fix the formatting? If so, please rename BindValues as bindValues()

Reply:
I changed from isColEncrypted to IsAEConnect on line 38 :)
Yes, I'll change to bindValues()